### PR TITLE
Add helper binding for task progress

### PR DESCRIPTION
**Pull request by fviale**
used with progress.set() in workflow tasks

Also fixed some paths in docker on windows fork environment

### CODE REVIEW

File: scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+ * @author ActiveEon Team
+ * @since 24/10/2019
+ */
+public class TaskProgressImpl implements TaskProgress {
+
+    File progressFile;
+
+    transient PrintStream output;
+
+    transient PrintStream error;
```
Comment (MaelAudren) [inline scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java]: VisibleForTesting ?

File: scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    public TaskProgressImpl(String progressFilePath, PrintStream output, PrintStream error) {
+        if (progressFilePath != null) {
+            progressFile = new File(progressFilePath);
+        } else {
+
+        }
```
Comment (MaelAudren) [inline scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java]: else empty ?

File: scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java (hunk 1 above)
Comment (fviale) [reply scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java]: we don't use VisibleForTesting in the scheduling project

File: scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java (hunk 2 above)
Comment (fviale) [reply scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java]: right it's a mistake

Comment (nebilbenmabrouk) [reply scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java]: else {
        throw new IllegalArgumentException("Invalid path for the progress file: " + progressFilePath);
}

File: scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java (hunk 1 above)
Comment (MaelAudren) [reply scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java]: ok

File: scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java (hunk 2 above)
Comment (fviale) [reply scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/TaskProgressImpl.java]: actually, I've removed the else case, this path could eventually be null in tests (in which case we do not care).

**Comment by fviale** [comment]
@nebilbenmabrouk this class is not called internally. It can be called by a workflow task using:

progress.set(23)

Basically, it's a binding inside workflow tasks

**Pull request merged**